src/app/pages/home/home.page.ts

```diff
@@ -178,7 +174,19 @@ export class HomePage extends RouterPage implements OnInit, AfterViewChecked, On
   }
 
   private createRegistrations$(searchCriteria$: Observable<SearchCriteria>): Observable<AtAGlanceViewModel[]> {
-    return combineLatest([searchCriteria$, this.refreshRequested$.pipe(startWith(true))]).pipe(
+    return combineLatest([
+      searchCriteria$,
+      this.tabsService.selectedTab$,
+      this.userSettingService.showObservations$,
+      this.refreshRequested$.pipe(startWith(true)),
+    ]).pipe(
+      filter(([, , showObservations]) => {
```

Comment:
Hvorfor bruke filter til dette? Filter brukes typisk til å filtrere bort noe fra en liste, feks.

Reply:
Hvis du endrer denne til en tap, så kan den godt heller ligge direkte på `this.userSettingService.showObservations$`, så den ikke kjøres unødvendig når feks searchCriteria$ endres.

Reply:
Valgte å fortsatt bruke filter, men på en annen observable.